src/Hydrators/TeamHydrator.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace TheRealMVP\Hydrators;
+use TheRealMVP\Entities\Team;
+
+class TeamHydrator 
+{
+    private \PDO $pdoConnection;
+
+    /**
+     * TeamHydrator constructor - saves PDO connection object as local variable
+     *
+     * @param $pdoConnection
+     */
+    public function __construct(\PDO $pdoConnection)
+    {
+        $this->pdoConnection = $pdoConnection;
+    }
+
+    /**
+     * create database connection and retrieves data and returns an array of team objects
+     *
+     * @param $pdoConnection
+     *
+     * @return array
+     */
+    public static function getData(\PDO $pdoConnection) : array
```

Comment:
Please remove the space before the : so it's consistent with everything else

This is also the recommended format according to psr-12 (https://www.php-fig.org/psr/psr-12/)